packages/cli/lib/services/compile.service.ts

```diff
@@ -89,5 +86,29 @@ class CompileService {
     }
 }
 
+export interface ListedFile {
+    inputPath: string;
+    outputPath: string;
+    baseName: string;
+}
+
+export function listFile(filePath: string): ListedFile {
+    if (!filePath.startsWith('./')) {
+        filePath = `./${filePath}`;
+    }
+    return {
+        inputPath: filePath,
+        outputPath: filePath.replace(/\/[^/]*$/, `/dist/${path.basename(filePath.replace('.ts', '.js'))}`),
+        baseName: path.basename(filePath, '.ts')
+    };
+}
+export function listFiles({ cwd, syncName }: { cwd?: string; syncName?: string | undefined }): ListedFile[] {
```

Comment:
This should have a more specific name in that it lists only typescript files.

Reply:
changed 👌🏻